src/openvr/OpenVRInterface.cpp

```diff
@@ -219,7 +215,9 @@ void OpenVRInterface::commit_for_eye(ARVRInterface::Eyes p_eye, RID p_render_tar
 
 				// TODO: THIS LOOP CAUSES PROBLEMS.
 				for (int i = 0; i < arvr_data->ovr->get_overlay_count(); i++) {
-					vr::TextureID_t texidov = (vr::TextureID_t)VisualServer::get_singleton()->texture_get_texid(VisualServer::get_singleton()->viewport_get_texture(arvr_data->ovr->get_overlay(i).viewport_rid));
+					vr::TextureID_t texidov = 0;
+						//DO things here
```

Comment:
another spot where I need to get the texid.

Reply:
Haven't even looked at overlays :)